Tests/Docker/Container/NetworkTest.php

```diff
@@ -70,61 +41,4 @@ public function testNetworkNone()
             self::assertStringContainsString('ping: www.example.com: Temporary failure in name resolution', $e->getMessage());
         }
     }
-
-    public function testNetworkBridgeOverride()
-    {
-        $this->setComponentConfig(['runtime' => ['network' => 'none']]);
-        $container = $this->getContainer($this->getBuilderImageConfiguration(), [], [], true);
-        try {
-            $container->run();
-            self::fail('Ping must fail');
-        } catch (UserException $e) {
-            self::assertStringContainsString('ping: www.example.com: Temporary failure in name resolution', $e->getMessage());
-        }
-    }
-
-    public function testNetworkBridgeOverrideNoValue()
-    {
-        $this->setComponentConfig(['runtime' => []]);
-        $container = $this->getContainer($this->getBuilderImageConfiguration(), [], [], true);
-        $process = $container->run();
-        self::assertEquals(0, $process->getExitCode());
-        self::assertStringContainsString('64 bytes from', $process->getOutput());
-    }
-
-    public function testNetworkBridgeOverrideFail()
-    {
-        $imageConfiguration = $this->getBuilderImageConfiguration();
-        $imageConfiguration['data']['definition']['build_options']['parameters'] = [];
-        $this->setComponentConfig(['runtime' => ['network' => 'none']]);
-        $container = $this->getContainer($imageConfiguration, [], [], true);
-        // parameter is not defined in image, must be ignored
-        $process = $container->run();
-        self::assertEquals(0, $process->getExitCode());
-        self::assertStringContainsString('64 bytes from', $process->getOutput());
-    }
-
-    public function testNetworkNoneOverride()
-    {
-        $imageConfiguration = $this->getBuilderImageConfiguration();
-        $imageConfiguration['data']['network'] = 'none';
-        $this->setComponentConfig(['runtime' => ['network' => 'bridge']]);
-        $container = $this->getContainer($imageConfiguration, [], [], true);
-        $process = $container->run();
-        self::assertEquals(0, $process->getExitCode());
-        self::assertStringContainsString('64 bytes from', $process->getOutput());
-    }
-
-    public function testNetworkInvalidOverride()
-    {
-        $imageConfiguration = $this->getBuilderImageConfiguration();
-        $imageConfiguration['data']['network'] = 'none';
-        $this->setComponentConfig(['runtime' => ['network' => 'fooBar']]);
-        try {
-            $this->getContainer($imageConfiguration, [], [], true);
-            self::fail('Invalid network must fail.');
-        } catch (ApplicationException $e) {
-            self::assertStringContainsString('not supported', $e->getMessage());
-        }
-    }
```

Comment:
Proc tyhle testu uz nejsou potreba? `network` uz nejde overridnout `runtime` konfiguraci? Resp. kdyz koukam na https://github.com/keboola/job-queue/blob/20f7a0294a5cb601cf1b75a9dece58798cc97fc0/libs/internal-api-php-client/src/JobFactory/JobRuntimeResolver.php tak mi prijde, ze to nikdy neslo :slightly_smiling_face: , tak nevim, jestli mi neco neunika

Reply:
psal jsem to v úvodu, šlo to overridnout jen pro builder komponenty. 

Jinak to overridnout opravdu nejde (logika za tím byla taková, že komponenta co má network none garantuje, že tvoje data nikam nepošle a může zpracovávat třeba PII a tímpádem ani nedává smysl, aby to overridnout šlo) - na tom se nic nemění